Content.Shared/Alert/AlertsComponent.cs

```diff
@@ -1,17 +1,27 @@
 using Robust.Shared.GameStates;
+using Robust.Shared.Serialization;
 
 namespace Content.Shared.Alert;
 
 /// <summary>
 ///     Handles the icons on the right side of the screen.
 ///     Should only be used for player-controlled entities.
 /// </summary>
-[RegisterComponent, NetworkedComponent, AutoGenerateComponentState(true)]
+[RegisterComponent, NetworkedComponent]
 public sealed partial class AlertsComponent : Component
 {
     [ViewVariables]
-    [AutoNetworkedField]
     public Dictionary<AlertKey, AlertState> Alerts = new();
```

Comment:
Should probably add an explanatory comment as for why this component is not autonetworked